Documentation/security/policy/kubernetes.rst

```diff
@@ -48,6 +48,47 @@ Using namespace-specific information like
 and not a `CiliumNetworkPolicy`. Hence, ``io.cilium.k8s.namespace.labels``
 will be ignored in `CiliumNetworkPolicy` resources.
 
+When using ``matchExpressions`` in a `CiliumNetworkPolicy` or 
+`CiliumClusterwideNetworkPolicy` the list values are
```

Comment:
I'd rather have the references explicitly required to avoid any confusion:

```suggestion
When using ``matchExpressions`` in a :ref:`CiliumNetworkPolicy` or a
:ref:`CiliumClusterwideNetworkPolicy`, the list values are
```

(Feel free to add the missing one to the previous paragraph, as well.)